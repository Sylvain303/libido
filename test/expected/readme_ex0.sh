#!/bin/bash 
#
# libido: bash_code=bash(die)
#
# Howto Rebuild: rebuild this code with:
# $ libido examples/readme_ex0.sh

echo 'some code here'

# libido code will be included/ updated here
# libido: expand bash_code
# expanded from: bash_code => die
die() {
    echo "$*"
    exit 1
}
echo 'some more code using die() and or docopts'
if [[ -z "$1" ]]
then
    die "missing argument1"
fi
exit 0
