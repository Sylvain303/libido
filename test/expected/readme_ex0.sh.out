some code here
some more code using die() and or docopts
missing argument1
